app/unit-tests/src/org/commcare/utils/FileUtilTest.java

```diff
@@ -1,32 +0,0 @@
-package org.commcare.utils;
-
-import org.commcare.CommCareTestApplication;
-import org.commcare.util.Base64;
-import org.commcare.util.Base64DecoderException;
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.robolectric.RobolectricTestRunner;
-import org.robolectric.annotation.Config;
-
-import java.io.UnsupportedEncodingException;
-
-import static junit.framework.TestCase.assertTrue;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import androidx.test.ext.junit.runners.AndroidJUnit4;
-
-/**
- * @author Clayton Sims (csims@dimagi.com)
- */
-@Config(application = CommCareTestApplication.class)
-@RunWith(AndroidJUnit4.class)
-public class FileUtilTest {
-
-    @Test
-    public void testContentTools() throws UnsupportedEncodingException, Base64DecoderException {
```

Comment:
why was it removed ?

Reply:
I removed it in this PR #2464 but somehow it got back. I moved this test here https://github.com/dimagi/commcare-android/blob/master/app/unit-tests/src/org/commcare/utils/FileUtilTest.kt#L45-L49 while changing to kotlin file.

Reply:
Sounds good